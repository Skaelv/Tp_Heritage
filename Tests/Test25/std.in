C C1 10 13 24
MOVE C1 0 0
EXIT
