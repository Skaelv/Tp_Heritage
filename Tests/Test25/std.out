 # Entrez une commande
OK
# New Object : C1
 # Entrez une commande
ERR
 #Invalid Parameters
 # Entrez une commande

